<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSudokuClass</class>
 <widget class="QMainWindow" name="QSudokuClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSudoku</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="mFile">
    <property name="title">
     <string>檔案</string>
    </property>
    <addaction name="aNewGame"/>
    <addaction name="aHintOneStep"/>
    <addaction name="separator"/>
    <addaction name="aExit"/>
   </widget>
   <widget class="QMenu" name="mConfig">
    <property name="title">
     <string>設定</string>
    </property>
    <addaction name="aBigBoard"/>
    <addaction name="aSmallBoard"/>
   </widget>
   <addaction name="mFile"/>
   <addaction name="mConfig"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="aExit">
   <property name="text">
    <string>離開</string>
   </property>
  </action>
  <action name="aNewGame">
   <property name="text">
    <string>開啟新局</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="aBigBoard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>大棋盤</string>
   </property>
  </action>
  <action name="aSmallBoard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>小棋盤</string>
   </property>
  </action>
  <action name="aHintOneStep">
   <property name="text">
    <string>提示一步</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QSudoku.qrc"/>
 </resources>
 <connections/>
</ui>
